useful-scripts/misc-test-utils/src/index.ts

```diff
@@ -1,17 +1,28 @@
+/* eslint-disable no-plusplus */
```

Comment:
Be good to see if there are valid ways to remove these warnings in a future iteration

Reply:
From looking at the eslint docs it seems it's just a matter of code style. i.e. 
```
var foo = 0;
foo += 1;

var bar = 42;
bar -= 1;

for (i = 0; i < l; i += 1) {
    return;
}
```
would not throw a lint error. I'll refactor stuff to use this style to stop the lint errors